frontend/actions/review.ts

```diff
@@ -34,11 +34,11 @@ type SemverOrAccessRequestId =
     }
   | {
       semver?: never
-      accessRequestId: AccessRequestInterface['id']
+      accessRequestId: AccessRequestInterface['id'] | undefined
```

Comment:
Use `?` to be consistent? e.g. `accessRequestId?: AccessRequestInterface['id'] `